lib/utils/aws/fips/fips_disabled.go

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-package aws
+package fips
```

Comment:
How do you feel about naming the package `awsfips` since that's what it'll always get imported as?

Reply:
Sure, give me a moment.

Reply:
Done!

Reply:
I meant calling the package `awsfips` with a path of `lib/utils/aws/fips` but this works too 😅

Reply:
Oh I see! I'd rather have the folder match, it's weird but less surprising.